pkg/cmd/agentversionjob.go

```diff
@@ -128,44 +59,51 @@ func (avj *AgentVersionCheckJob) Status() error {
 
 // Execute - run agent version check job one time
 func (avj *AgentVersionCheckJob) Execute() error {
-	err := avj.getJFrogVersions()
+	state, err := avj.getAgentState()
 	if err != nil {
 		log.Trace(err)
-		// Could not get update from jfrog.  Warn that we could not determine version and continue processing
-		log.Warn("Agent cannot determine the next available release. Be aware that your agent could be outdated.")
-	} else {
-		// Successfully got jfrog version.  Now compare build to latest version
-		if isVersionStringOlder(avj.buildVersion, config.AgentLatestVersion) {
-			log.Warnf("New version available. Please consider upgrading from version %s to version %s", avj.buildVersion, config.AgentLatestVersion)
-		}
+		// Could not get update from agent state.  Warn that we could not determine version and continue processing
+		log.Warn("Agent cannot determine the current available release. Be aware that your agent could be outdated.")
```

Comment:
maybe we should return after adding this warning

Reply:
I guess its 6 in one half dozen in the other but does avoid the switch statement

Reply:
Can I get a confirmation on case for logging at the start of the of the sentence? And does it matter what level? We have both upper and lower case here